util/common/pom.xml

```
@@ -37,5 +37,29 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <exports combine.self="override">
+                                        org.hibernate.search.*.impl to org.hibernate.search.util.common, org.hibernate.search.engine, org.hibernate.search.backend.elasticsearch, org.hibernate.search.backend.elasticsearch.aws, org.hibernate.search.mapper.pojo, org.hibernate.search.mapper.javabean, org.hibernate.search.mapper.orm, org.hibernate.search.batch.jsr352.core, org.jboss.logging;
+                                        *;
```

Why is this not necessary, while it is apparently necessary for other modules?

```suggestion
                                        !org.hibernate.search.*.impl;
                                        *;
```

---

Actually It is not necessary, because the exact same expression `org.hibernate.search.*.impl` is exported to a set of modules.
I'm going to remove it for the other modules too!